File: app/src/main/java/chenmc/sms/data/VerificationCodeSms.java
package chenmc.sms.data;

/**
 * 验证码短信
 * Created by 明明 on 2017/7/1.
 */

public class VerificationCodeSms extends CodeSms {
    public VerificationCodeSms() {
    }
    
    public VerificationCodeSms(String code) {
        super(code);
    }
    
    public VerificationCodeSms(String serviceProvider, String verificationCode) {
        super(serviceProvider, verificationCode);
    }
    
    public VerificationCodeSms(String serviceProvider, String code, String content) {
        super(serviceProvider, code, content);
    }
}

File: app/src/main/java/chenmc/sms/data/VerificationCodeSms.kt
package chenmc.sms.data

/**
 * 验证码短信
 * Created by 明明 on 2017/7/1.
 */

class VerificationCodeSms @JvmOverloads constructor(
    raw: String,
    serviceProvider: String = "",
    code: String = "",
    extra: String = ""
) : CodeSms(raw, serviceProvider, code, extra)